angular-ngrx-scss/src/app/user/services/user.service.spec.ts

```diff
@@ -55,4 +62,182 @@ describe('UserService', () => {
+        name: 'Repo-test-2',
+        description: 'This is a repo test 2',
+        language: 'Javascript',
+        license: null,
+        private: false,
+        stargazers_count: 0,
+        forks_count: 0,
+        updated_at: '2022-06-17T09:54:38Z',
+        owner: {
+          avatar_url: 'https://avatars.githubusercontent.com/u/22839396?v=4',
+          events_url: 'https://api.github.com/users/thisdot/events{/privacy}',
+          followers_url: 'https://api.github.com/users/thisdot/followers',
+          following_url:
+            'https://api.github.com/users/thisdot/following{/other_user}',
+          gists_url: 'https://api.github.com/users/thisdot/gists{/gist_id}',
+          gravatar_id: '',
+          html_url: 'https://github.com/thisdot',
+          id: 22839396,
+          login: 'thisdot',
+          node_id: 'MDEyOk9yZ2FuaXphdGlvbjIyODM5Mzk2',
+          organizations_url: 'https://api.github.com/users/thisdot/orgs',
+          received_events_url:
+            'https://api.github.com/users/thisdot/received_events',
+          repos_url: 'https://api.github.com/users/thisdot/repos',
+          site_admin: false,
+          starred_url:
+            'https://api.github.com/users/thisdot/starred{/owner}{/repo}',
+          subscriptions_url:
+            'https://api.github.com/users/thisdot/subscriptions',
+          type: 'Organization',
+          url: 'https://api.github.com/users/thisdot',
+        },
+      },
+      {
+        name: 'Repo-test-3',
+        description: 'This is a repo test 2',
+        language: 'Javascript',
+        private: false,
+        stargazers_count: 0,
+        forks_count: 0,
+        updated_at: '2022-06-17T09:54:38Z',
+        owner: {
+          avatar_url: 'https://avatars.githubusercontent.com/u/22839396?v=4',
+          events_url: 'https://api.github.com/users/thisdot/events{/privacy}',
+          followers_url: 'https://api.github.com/users/thisdot/followers',
+          following_url:
+            'https://api.github.com/users/thisdot/following{/other_user}',
+          gists_url: 'https://api.github.com/users/thisdot/gists{/gist_id}',
+          gravatar_id: '',
+          html_url: 'https://github.com/thisdot',
+          id: 22839396,
+          login: 'thisdot',
+          node_id: 'MDEyOk9yZ2FuaXphdGlvbjIyODM5Mzk2',
+          organizations_url: 'https://api.github.com/users/thisdot/orgs',
+          received_events_url:
+            'https://api.github.com/users/thisdot/received_events',
+          repos_url: 'https://api.github.com/users/thisdot/repos',
+          site_admin: false,
+          starred_url:
+            'https://api.github.com/users/thisdot/starred{/owner}{/repo}',
+          subscriptions_url:
+            'https://api.github.com/users/thisdot/subscriptions',
+          type: 'Organization',
+          url: 'https://api.github.com/users/thisdot',
+        },
+      },
+    ];
+    httpClientSpy.get.and.returnValue(of(expectedHttpResponse).pipe(delay(0)));
```

Comment:
question: Why is this `.pipe(delay(0))` needed?

Reply:
Removed, since now the observables are handled from the fakeAsync approach.